subt/ros/robot/launch/k2.launch

```diff
@@ -1,96 +1,92 @@
+<?xml version="1.0"?>
 <launch>
-	<env name="DISPLAY" value=":0.0"/>
-	<!-- LOGGING -->
-	<env name="ROSCONSOLE_CONFIG_FILE"  value="$(find robot)/custom_rosconsole.conf"/>
-        <arg name="record_sensors" default="true" />
-        <!--node pkg="rosbag" type="record" name="rosbag_record" args="record -o /home/md/ros_ws/bags/ /tf /camera/color/image_raw/compressed /camera/color/camera_info /rs_t265/odom/sample /camera/depth/image_rect_raw /camera/depth/camera_info /wheel_odom  __name:=my_bag" if="$(arg record_sensors)"/-->
-        <node pkg="rosbag" type="play" name="player" output="screen" args="--pause --clock /home/pavliik/develop/ros_ws/bags/_2020-02-14-00-35-16.bag" unless="$(arg record_sensors)"/>
-	<group unless="$(arg record_sensors)">
-		<param name="use_sim_time" value="true"/>
-	</group>
-	<group if="$(arg record_sensors)">
-                <param name="use_sim_time" value="false"/>
-        </group>
+  <!-- Announces position of the point tracked by tracking camera relative to the center of the robot. -->
+  <node pkg="tf" type="static_transform_publisher" name="tracked_point_tf" args="0 0 0 0 0 0 global odom 100" />
+  <!-- Announces the position of the center of the robot relative to the point tracked by tracking camera in Osgar.
+       The tracked point name needs to match the name of the stream sent from Osgar to ROS. -->
+  <node pkg="tf" type="static_transform_publisher" name="robot_tf" args="0 0 0 0 0 0 tracking_pose robot 100" />
 
-	<node name="map_to_scan" pkg="robot" type="map_to_scan.py"  output="screen"/>
+  <!-- Announces the position of depth camera relative to the center of the robot.
+       The name of the announced frame needs to match the name of the corresponding depth stream from Osgar to ROS extended with a "/pose" suffix to distinguish from related optical frames configured below. -->
+  <node pkg="tf" type="static_transform_publisher" name="front_camera_tf" args="0.5 0 0.525 0 -0.4363323 0 robot front_depth/pose 100" />
+  <node pkg="tf" type="static_transform_publisher" name="rear_camera_tf" args="-0.5 0 0.59 -3.141592653589793 -0.4363323 0 angle rear_depth/pose 100" />
```

Comment:
The `angle` frame name in here must match the name of the channel sent from Osgar to the ROS pull node, because the pull node uses the channel name as the name of the published frame. I.e. after your recent changes, you need `joint_angle_pose` instead of 'angle`.

Reply:
Sorry, you are right, you have `["from_apu_receiver.joint_angle_pose", "toros.angle"]` in the front json. This configuration looks OK.